profile/README.md

```diff
@@ -1,11 +1,76 @@
-# Hi There 👋 --- Welcome to our Organization's GitHub Page
+# Nautilus Cyberneering Github Organization
 
-![Open Source](https://badgen.net/badge/Open%20Source/100%25/DA2CE7)
-![Cool](https://badgen.net/badge/Cool/100%25/FF7F50)
-![Awesome](https://badgen.net/badge/icon/awesome?icon=awesome&label)
+Logo
+
+Badge Nautilus Website, Badge Torrust Website.
+
+## Welcome
+
+Nautilus Cyberneering SL, an upstart subsidiary of the Block Finance AG group, has the mission cultivate an open source software engineering and development team. This will provide strategic and practical advantages for the group.
+
+## History
+
+Originally started as Nautilus Cyberneering GmbH and based out of Germany, Block Finance AG wholly purchased the company and liquidate the original company structure, transferring all intellectual property and other rights to the group.
+
+Simultaneously, the group took inspiration of the business concept that Nautilus Cyberneering GmbH had and decided to re-create a similar company in the more favorable jurisdiction and climate of Las Palmas, Grand Canaria; thus Nautilus Cyberneering SL was born, the direct spiritual successor from Nautilus Cyberneering GmbH.
+
+## Goals
+
+Block Finance AG has significant need for in-group world-class computer science engineering and development capabilities, and is duly aware that establishing such capabilities in beautiful running and vibrant professional team quality is an non-trivial endeavour.
+
+Nautilus adopts the strategy of focusing sponsoring and facilitating the development of open-source projects as a way to gain real-world experience of software engineering, project management, and team building. The projects we focus on along with the associated languages and technologies, have been selected with the goal of being attractive for competent and passionate software engineers.
+
+The secondary goal of our sponsorship and facilitation of interesting open-source projects is that these projects may become successful in their own right; growing a vibrant and active community around them; positioning Nautilus with additional exposure and access to a sympathetic market of talented software engineers, developers, architects, testers, administrators, community focalises, and supportive end users.
+
+Nautilus success is to find itself surrounded with pools of brilliant talent that have accumulated from the natural community growth that accompanies healthy open-source projects. The entire Block Finance Group can draw from these well connect pools; giving the group access to great talent, and the community access to friendly professional employment and sponsorship opportunities.
+
+## Ethos
```

Comment:
I find the choice of word Ethos is not appropriate in relation to a company in what you want to express. In relation to a company you normally speak of culture. 

I was curious as to the difference that may exist in case I was not aware of it I checked with Chat GPT asking for the difference between the two in relation to a company and it gave me this answer which I include for you:

In the context of a company, "ethos" and "culture" are related but distinct concepts that collectively shape the character, values, and behaviors of the organization. Here are the key differences between them:

1. **Ethos**:
   - **Individual Focus**: Ethos often refers to the character or guiding beliefs and principles of an individual or a smaller group within the organization, such as its leadership or founders.
   - **Personal Values**: It is more focused on the personal values, ethics, and philosophies of individuals or specific leaders that influence decision-making and behavior within the company.
   - **Top-Down Influence**: Ethos is typically set and influenced from the top leadership of the organization and can serve as a model for others to follow.
   - **Can Vary**: Ethos can vary among different individuals or groups within the company, especially in larger organizations.

2. **Culture**:
   - **Collective Focus**: Culture encompasses the shared values, beliefs, attitudes, and behaviors that permeate the entire organization, affecting how all employees interact and work together.
   - **Organizational Values**: It reflects the core values and principles that guide how the company operates, makes decisions, and treats its employees, customers, and stakeholders.
   - **Widespread Influence**: Culture is pervasive and influences all levels of the organization, shaping how employees work, communicate, and make decisions.
   - **Unified**: Ideally, company culture seeks to create a unified and cohesive environment where everyone aligns with a common set of values and behaviors.

In summary, while ethos can be seen as a subset of an organization's culture, it is more specific and tends to focus on the personal values and beliefs of influential individuals within the company. Culture, on the other hand, is a broader concept that encompasses the collective values, behaviors, and norms that define the entire organization's identity and how it functions as a whole. Both ethos and culture play important roles in shaping the overall character and identity of a company.

I think that after reading this I would go for company culture, since we are a group of people, more than one company who share a common set of values, and core beliefs.